ui/admin/app/components/form/storage-bucket/aws/index.hbs

```diff
@@ -455,13 +455,14 @@
   {{! worker_filter }}
   <Hds::Form::Fieldset
     class='storage-bucket-worker-filter'
+    aria-labelledby='worker-filter-label worker-filter-help'
     @isRequired={{true}}
     as |F|
   >
-    <F.Legend>{{t
+    <F.Legend id='worker-filter-label'>{{t
```

Comment:
i saw one component using the `(unique-id)` helper but others using a simple string that is unique. Let me know if I should change my approach here.

Reply:
thank you for the suggestion, updated!
Refactored to use the generateComponentID helper that uses `uuid`

Reply:
I don't have an issue with using generated unique IDs but we should probably use `(unique-id)` like you mentioned. I'm pretty sure `generateComponentID` predated ember adding [unique-id](https://api.emberjs.com/ember/5.12/classes/Ember.Templates.helpers/methods/unique-id?anchor=unique-id) but now that they provide it as a built in helper we should use it.

Reply:
ah good point, ill refactor to use `unique-id`! Thanks for providing the link!